contracts/CompoundGovernor.sol

```diff
@@ -64,6 +114,69 @@ contract CompoundGovernor is
         __GovernorPreventLateQuorum_init(_initialVoteExtension);
         __GovernorSettableFixedQuorum_init(_quorumVotes);
         __Ownable_init(_initialOwner);
+        _setWhitelistGuardian(_whitelistGuardian);
+        _setProposalGuardian(_proposalGuardian);
+    }
+
+    /// @notice Sets or updates the whitelist expiration for a specific account.
+    /// @param _account The address of the account to be whitelisted.
+    /// @param _expiration The timestamp until which the account will be whitelisted.
+    /// @dev Only the executor (timelock) or the whitelistGuardian can call this function.
+    function setWhitelistAccountExpiration(address _account, uint256 _expiration) external {
```

Comment:
Naming nit: i like the specificity of this method name, i just wish it captured more of the main intent which is to activate/deactivate a whitelisted account. are there other names that might work better? something like `whitelistAccount(address, uint256)` with an event `WhitelistedAccountSet` or something?

Reply:
We agreed to keep the old naming to reduce the scope of changes made to the contract.